internal/web/server.go

```diff
@@ -18,6 +18,12 @@ import (
 	"sync"
 	"syscall"
 
+	types2 "k8s.io/apimachinery/pkg/types"
```

Comment:
I think a better name conflict resolution would be changing the import name of "repo/types" to `repotypes` and leaving this one as `types` or maybe `apitypes`